test/api-services/foundations-api.service-specs.js

```diff
@@ -1,3 +1,4 @@
+/* global beforeEach */
```

Comment:
You can set the mocha environment setting in the eslint config to avoid having to do this: https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
good to know, this applies to other code as well